proto/greenfield/storage/types.proto

```diff
@@ -87,6 +87,14 @@ message ObjectInfo {
   repeated bytes checksums = 14 [(gogoproto.moretags) = "traits:\"omit\""];
   // tags defines a list of tags the object has
   ResourceTags tags = 15;
+  // is_updating indicates whether a object is being updated.
+  bool is_updating = 16;
+  // updated_at define the block timestamp when the object is updated. Will not be visible until object is re-sealed.
+  int64 updated_at = 17;
+  // updated_by defined the account address of updater(if there is). Will not be visible until object is re-sealed.
+  string updated_by = 18 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
Does `updated_by` matter? Because when we upload a object, we did not have a field called `uploaded_by`..